controllers/dscinitialization/suite_test.go

```diff
@@ -62,18 +89,51 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(cfg).NotTo(BeNil())
 
-	err = dscinitializationv1.AddToScheme(scheme.Scheme)
-	Expect(err).NotTo(HaveOccurred())
-
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme.Scheme))
+	utilruntime.Must(dscinitializationv1.AddToScheme(scheme.Scheme))
+	utilruntime.Must(netv1.AddToScheme(scheme.Scheme))
+	utilruntime.Must(authv1.AddToScheme(scheme.Scheme))
+	utilruntime.Must(corev1.AddToScheme(scheme.Scheme))
+	utilruntime.Must(apiextv1.AddToScheme(scheme.Scheme))
+	utilruntime.Must(routev1.AddToScheme(scheme.Scheme))
+	utilruntime.Must(appsv1.AddToScheme(scheme.Scheme))
+	utilruntime.Must(ofapi.AddToScheme(scheme.Scheme))
 	//+kubebuilder:scaffold:scheme
 
+	// Initialize Kubernetes client
```

Comment:
```suggestion
```

I mean... you can see `client.New` and you know that's what is happening :) No need for extra comment here in my view. I would strongly vouch for commenting about "whys" in the code, but "whats" are most of the time obvious from just reading the code :)